scri/SpEC/file_io/__init__.py

```diff
@@ -604,10 +604,14 @@ def create_abd_from_h5(
         if file_format == "spectrecce_v1":
             WMs[i].t -= float(radius)
 
+        indices = index_is_monotonic(WMs[i].t)
+        WMs[i].t = WMs[i].t[indices]
+        WMs[i].data = WMs[i].data[indices]
+            
     # Create an instance of AsymptoticBondiData
     abd = AsymptoticBondiData(
-        time=WM_ref.t,
-        ell_max=WM_ref.ell_max,
+        time=WMs[list(WMs.keys())[0]].t,
+        ell_max=WMs[list(WMs.keys())[0]].ell_max,
```

Comment:
Why not still just use `WM_ref` here?  No copy was made, so `WM_ref` should have been changed in the same way, right?

Reply:
Good point; reverted back!